packages/liveblocks-yjs/src/index.ts

```diff
@@ -236,6 +245,10 @@ export class LiveblocksYjsProvider<
     }
   };
 
+  get useV2Encoding(): boolean {
+    return this.options.useV2Encoding_experimental ?? false;
+  }
```

Comment:
I suppose this getter is required for reason no? Will it be reuse by some other objects?

Reply:
Not really, it originally had more logic earlier in my development cycle, now it's mostly vestigial/convenience.